cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,226 @@
+                cy.get("#merge_to_11_districts_checkbox").should("be.checked");
+                cy.get("#2021_distribution_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("2021 distribution checkbox, then merge checkbox", () => {
+        it("Works for relevant years", () => {
+            cy.visit("").then(() => waitForLoad());
+
+            yearsWithModernElectionSystem.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                cy.get("#2021_distribution_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").should("be.checked");
+                cy.get("#2021_distribution_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("Algorithms", () => {
+        years.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const algorithms = [
+                    "SAINTE_LAGUE",
+                    "D_HONDT",
+                    "LARGEST_FRACTION_HARE",
+                    "LARGEST_FRACTION_DROOP",
+                    "LARGEST_FRACTION_HAGENBACH_BISCHOFF",
+                ];
+                algorithms.forEach((algorithm) => {
+                    cy.get("#algorithm_select").select(algorithm);
+                    cy.get("#algorithm_select").should("have.value", algorithm);
+                });
+            });
+        });
+    });
+
+    context("First divisor", () => {
+        yearsWithStLagues.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const divisorValues = ["1", "2", "3", "4", "5"];
+                divisorValues.forEach((value) => {
+                    cy.get("#firstDivisor").clear();
+                    cy.get("#firstDivisor").type(value);
+                    cy.get("#firstDivisor").should("have.value", value);
+                });
+            });
+        });
+    });
+
+    context("Leveling seat threshold", () => {
+        years.forEach((year) => {
+            it(`Works for ${year}`, () => {
+                cy.visit("").then(() => waitForLoad());
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const electionThresholdValues = ["0", "5", "7.5", "10", "15"];
+                electionThresholdValues.forEach((electionThreshold) => {
+                    cy.get("#electionThreshold").clear();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)